common/predictive-text/worker/index.ts

```diff
@@ -349,7 +349,7 @@ if (typeof module !== 'undefined' && typeof module.exports !== 'undefined') {
   module.exports['ModelCompositor'] = ModelCompositor;
 } else if (typeof self !== 'undefined' && 'postMessage' in self) {
   // Automatically install if we're in a Web Worker.
-  LMLayerWorker.install(self as DedicatedWorkerGlobalScope);
+  LMLayerWorker.install(self as any); // really, 'as typeof globalThis', but we're currently getting TS errors from use of that.
```

Comment:
This line only makes sense in the context when we _know_ we have the DOM. This is the `.install()` invocation that runs in an actual web worker. Can this line be put in the compilation configuration that assumes that the DOM exists?

Reply:
... actually, this line is (currently) used quite effectively for headless workers too.  Even in a "virtualized" `Worker`, we'll still want to install to the (virtualized) global scope.